app/tasks/usage.php

```diff
@@ -487,7 +487,7 @@
                                 'files' => [
                                     'namespace' => '',
                                     'collectionPrefix' => 'bucket_',
-                                    'sum' => [
+                                    'total' => [
```

Comment:
This is most likely forcing changes in the UI phtml files.

Reply:
Any specific place? I just updated UI, I think I found all sums.